diff --git a/node_modules/@vscode/windows-ca-certs/binding.gyp b/node_modules/@vscode/windows-ca-certs/binding.gyp
index 1f7f13f..a74e5d5 100644
--- a/node_modules/@vscode/windows-ca-certs/binding.gyp
+++ b/node_modules/@vscode/windows-ca-certs/binding.gyp
@@ -17,9 +17,6 @@
       "link_settings": {
         "libraries": ["-lcrypt32"]
       },
-      "msvs_configuration_attributes": {
-        "SpectreMitigation": "Spectre"
-      },
       "msvs_settings": {
         "VCCLCompilerTool": {
           "AdditionalOptions": [
